sensu/files/plugins/graphite-data.rb

```diff
@@ -192,7 +192,7 @@ def retrieve_data
   # Return alert if required
   def check(type)
     if config[type]
-      send(type, "#{name} (#{value_to_check(@data)}) [#{@value['target']}]") if (below?(type) || above?(type))
+      send(type, "#{name} (#{value_to_check(@data)}) [#{@value['target']}]") if (in_alert_range?(type))
```

Comment:
Ruby style niggle: `if in_alert_range?(type)` (i.e. remove the parenthesis around the condition for a postifxed condition.